digdag-docs/src/operators.rst

```diff
@@ -880,3 +880,97 @@ embulk>: Embulk data transfer
 
   * :command:`embulk>: embulk/mysql_to_csv.yml`
 
+
+s3_wait>: Wait for a file in Amazon S3
+--------------------------------------
+
+The **s3_wait>:** operator waits for file to appear in Amazon S3.
+
+.. code-block:: yaml
+
+    +wait:
+      s3_wait>: my-bucket/my-key
+
+Secrets
+~~~~~~~
+
+:command:`aws.s3.access-key, aws.access-key`
+  The AWS Access Key ID to use when accessing S3.
+
+:command:`aws.s3.secret-key, aws.secret-key`
+  The AWS Secret Access Key when accessing S3.
+
+:command:`aws.s3.region, aws.region`
+  An optional explicit AWS Region in which to access S3.
+
+:command:`aws.s3.endpoint`
+  An optional explicit API endpoint to use when accessing S3. This overrides the `aws.region` secret.
```

Comment:
It actually overrides the region secret.

If no endpoint is configured, the s3 client uses an endpoint based on the region specified. Explicitly setting an endpoint overrides this.

Still, I should probably change that from `aws.region` to just `region`.